server/bleep/src/indexes/file.rs

```diff
@@ -167,7 +167,7 @@ impl Indexable for File {
         repo: &Repository,
         repo_info: &RepoHeadInfo,
         writer: &IndexWriter,
-    ) -> Result<()> {
+    ) -> Result<()> {        
```

Comment:
Empty spaces at the end of line!

Reply:
We need to get the CI back up. Have to configure qdrant builds: https://github.com/BloopAI/bloop/actions/runs/3902002219/jobs/6664534799

Reply:
Oops, fixed.